sdk/spring/azure-spring-cloud-autoconfigure/HEALTH_INDICATOR_README.md

```diff
@@ -0,0 +1,101 @@
+# Azure Spring Boot - Actuator
```

Comment:
Same for this one, how about we add these two HEALTH_INIDCATOR_README as a section to this [README.md](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/README.md).